reviews/templates/mail/concept_other_applicants.html

```diff
@@ -0,0 +1,23 @@
+{% extends "mail/base.html" %}
+
+{% block content_nl %}
+Beste collega,<br />
+<br />
+Uw collega {{ creator }} heeft een concept-aanmelding ingediend bij de ethische commissie voor een onderzoek, waar u aan meewerkt.<br />
```

Comment:
(This comment applies to all emails)

The UU writing styleguide prefers informal language over formal, in Dutch, for internal communication. 
(Ofwel, gebruik 'je' ipv 'u').
I see that not all mails were rewritten in the app, so imo this can be fixed separately as a part of a generic 'fix language' issue.  

For context, the passage about it in the writing styleguide:
> In interne communicatie spreken we medewerkers en studenten aan met ‘je’, ook op schrift. In externe communicatie hangt de aanspreekvorm af van de persoon en de context.